app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background-image: image_url("background1.jpg");
+  font-family: "Karma", serif;
+}
+
+h1 {
+  font-weight: bold;
+  color: #DF7401;
+}
+
+h2 {
+  font-weight: bold;
+  color: #FAAC58;
+}
+
+.top-bar {
+  margin-bottom: 10px;
+}
+
+.button {
+  margin-right: 10px;
+}
+
+.button a {
+  color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`